Detectors/MUON/MCH/Raw/Decoder/src/DataDecoder.cxx

```diff
@@ -715,6 +738,21 @@ void DataDecoder::init()
 
   initFee2SolarMapper(mMapCRUfile);
   initElec2DetMapper(mMapFECfile);
+
+  mTimeFrameStartRecords.resize(sReadoutChipsNum);
+  std::fill(mTimeFrameStartRecords.begin(), mTimeFrameStartRecords.end(), TimeFrameStartRecord());
+
+  mMergerRecords.resize(sReadoutChannelsNum);
+  mMergerRecordsReady.resize(sReadoutBoardsNum);
+
+  if (mMergerRecords.size() != DataDecoder::sReadoutChannelsNum) {
+    std::cout << "WRONG SIZE OF mMergerRecords" << std::endl;
```

Comment:
could you use `LOGP(error,".......")` instead of bare `cout` so that the messages are accessible in the info logger as well ?

Reply:
Those check are actually not really needed, just a leftover of some debugging. I have removed  them.